src/nodes/core/NodeBuilder.js

```diff
@@ -937,22 +1111,25 @@ class NodeBuilder {
 
 	}
 
-	getStructTypeFromNode( node, shaderStage = this.shaderStage ) {
+	getStructTypeFromNode( node, types, shaderStage = this.shaderStage ) {
```

Comment:
@sunag I have slightly refactored `getStructTypeFromNode()` so it is more similar to `getBufferAttributeFromNode()` or `getUniformFromNode()`.